pkg/child/child.go

```diff
@@ -583,13 +583,19 @@ func NewNetNsWithPathWithoutEnter(p string) error {
 	if err := os.WriteFile(p, nil, 0400); err != nil {
 		return err
 	}
-	tempNS, err := ns.TempNetNS()
+	selfExe, err := os.Executable()
 	if err != nil {
 		return err
 	}
-	defer tempNS.Close()
-	tempNSPath := tempNS.Path()
```

Comment:
@AkihiroSuda, I don't understand why `ns.WithNetNSPath` cannot detect the AppArmor profile configuration.

Reply:
Probably a race condition?

nerdctl CI was green without reverting this commit, but BuildKit CI was failing.